podman/api/parse_utils.py

```diff
@@ -95,6 +98,10 @@ def stream_frames(response: Response) -> Iterator[bytes]:
             continue
 
         data = response.raw.read(frame_length)
+
+        if demux:
+            data = demux_output(header + data)
+
```

Comment:
following from the comments below, is demux required here? I believe it can be checked in the exec_run function. or am I missing something

Reply:
If we want to imitate the docker sdk we need it here because the header contains whether the data is `STDOUT` or `STDERR` and we lose the header in the output.

Reply:
ok so, as before, now I understand the code and I am ok with the solution. but let's test it